packages/ripple-keypairs/test/codec.test.ts

```diff
@@ -43,11 +43,14 @@ describe('ripple-address-codec', function() {
     assert.equal(decoded2.type, 'secp256k1')
   })
 
-  it('can pass a type as second arg to encodeSeed', function() {
+  it('can pass a type as second arg to encodeSeed', () => {
     const edSeed = 'sEdTM1uX8pu2do5XvTnutH6HsouMaM2'
     const decoded = api.decodeSeed(edSeed)
     assert.equal(toHex(decoded.bytes), '4C3A1D213FBDFB14C7C28D609469B341')
     assert.equal(decoded.type, 'ed25519')
+    // @ts-expect-error -- We are testing ed25519 here
     assert.equal(api.encodeSeed(decoded.bytes, decoded.type), edSeed)
   })
 })
+
+export {}
```

Comment:
Why is this needed? It's a test file

Reply:
https://stackoverflow.com/a/71920196

Reply:
Shouldn't tests not be treated as isolated modules? Is it possible to only have that flag for the libraries themselves?

Reply:
I'm not sure, I can add to followups